pkgs/development/haskell-modules/configuration-common.nix

```
@@ -23,6 +23,12 @@ self: super: {
   # There are numerical tests on random data, that may fail occasionally
   lapack = dontCheck super.lapack;
 
+  # fix tests failure for base≥4.15
+  leveldb-haskell = appendPatch (fetchpatch {
+    url = "https://github.com/kim/leveldb-haskell/pull/41.patch";
```

Using the commit is a bit more robust as it wouldn't change content on a force push:

```suggestion
    url = "https://github.com/kim/leveldb-haskell/commit/f5249081f589233890ddb1945ec548ca9fb717cf.patch";
```

---

Uhm, it's true but also hides the fact the patch comes from a pull request. 
I put the link in the comment.